lib/rspec/core/formatters/documentation_formatter.rb

```diff
@@ -27,19 +32,40 @@ def example_group_finished(_notification)
 
         def example_passed(passed)
           output.puts passed_output(passed.example)
+          flush_messages
         end
 
         def example_pending(pending)
           output.puts pending_output(pending.example,
                                      pending.example.execution_result.pending_message)
+          flush_messages
         end
 
         def example_failed(failure)
           output.puts failure_output(failure.example)
+          flush_messages
+        end
+
+        def message(notification)
+          if @messages
+            @messages << notification.message
+          else
+            output.puts "#{current_indentation(1)}#{notification.message}"
```

Comment:
This should be `0` indentation

Reply:
I'm okay with this, but doesn't it make sense for the message to come out nested within the current grouping?